bubble-wrap.yaml

```diff
@@ -852,6 +878,37 @@ layers:
                 transition:
                     [show, hide]:
                         time: 0s
+                        
+    # Used by the SDK to place point, line, and polygon overlays on the map
+    mz_default_point:
+        data: { source: mz_default_point }
+        draw:
+            sdk-point-overlay:
+                interactive: true
+                sprite: search-active
+                size: [36px,54px]
+                collide: false
+                anchor: top
+                transition:
+                    [show, hide]:
+                        time: 0s
+    mz_default_line:
+        data: { source: mz_default_line }
+        draw:
+            sdk-line-overlay:
+                color: '#06a6d4'
+                order: 503
+                width: 3px
+    mz_default_polygon:
+        data: { source: mz_default_polygon }
+        draw:
+            sdk-polygon-overlay:
+                color: [0.02,0.65,0.82,0.5]  #'#06b1e2'
+                order: 501
+            sdk-line-overlay:
+                color: '#06a6d4'
+                order: 502
+                width: 0px
```

Comment:
I'm setting this to `0px` here to disable the stroke on the polygon, but if someone wanted to draw stroked polygons the'd set the width and it'd show up blue line by default. But it's not clear to me how the SDK would modify these values / if this is the best approach to take.